### PR TITLE
feat: add fields in photon metadata.json to support running custom image

### PR DESCRIPTION
**Pull request by bddppq**
```py
from leptonai import Photon


class CustomImage(Photon):
    image = "a:b"
    exposed_port = 8765
    cmd = ["python", "-m", "http.server", str(exposed_port)]
```

metadata.json:
```
{
...
  "image": "a:b",
  "exposed_port": 8765,
  "cmd": [
    "python",
    "-m",
    "http.server",
    "8765"
  ]
...
}
```

### CODE REVIEW

**Comment by xiang90** [comment]
cmd change looks good to me.

/cc @ccding can you confirm the health check path?

**Comment by ccding** [comment]
in k8s, the hc path is callde `LivenessProbe.HTTPGet.Path`. I think it is too long. We should simplify it to `health_check_path` (discussed with JJ earlier today)

**Comment by xiang90** [comment]
shall we at least add the http prefix?

**Comment by ccding** [comment]
yeah, makes senes

File: leptonai/photon/photon.py [hunk 1]
```diff
@@ -174,7 +176,10 @@ class Photon(BasePhoton):
     image: str = BASE_IMAGE
 
     # The args for the base image.
-    args: list = BASE_IMAGE_ARGS
+    args: List[str] = BASE_IMAGE_ARGS
+    cmd: Optional[List[str]] = BASE_IMAGE_CMD
```
Comment (Yangqing) [inline leptonai/photon/photon.py:180]: Question for @xiang90 @ccding - is it more common to call it "cmd" or "entry_point" in the cloud native context? Python doesn't really have a strong convention so I am kinda curious what name is the best.

**Pull request merged**